packages/zero-cache/src/scripts/run-query.ts

```diff
@@ -58,12 +58,47 @@ const host: QueryDelegate = {
   },
 };
 
-const pipeline = buildPipeline(ast, host);
-const output = new Catch(pipeline);
+let start: number;
+let end: number;
+const suppressError: Record<string, unknown> = {};
+if (config.debug.ast) {
+  [start, end] = runAst(JSON.parse(config.debug.ast) as AST);
+} else if (config.debug.query) {
+  [start, end] = runQuery(config.debug.query);
+} else {
+  throw new Error('No query or AST provided');
+}
+
+function runAst(ast: AST): [number, number] {
+  const pipeline = buildPipeline(ast, host);
+  const output = new Catch(pipeline);
 
-const start = performance.now();
-output.fetch();
-const end = performance.now();
+  const start = performance.now();
+  output.fetch();
+  const end = performance.now();
+  return [start, end];
+}
+
+function runQuery(queryString: string): [number, number] {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  let q: any;
+  const z = {
+    query: Object.fromEntries(
+      Object.entries(schemaAndPermissions.schema.tables).map(([name]) => [
+        name,
+        newQuery(host, schemaAndPermissions.schema, name),
+      ]),
+    ),
+  };
+  suppressError.q = q;
```

Comment:
Is this used for anything?

Reply:
TypeScript would throw errors about `q` not being used. Let me add a comment and I can change this to `@ts-expect-error`